qa/suites/rados/mgr/tasks/module_selftest.yaml

```diff
@@ -16,6 +16,7 @@ tasks:
         - objects misplaced
         - Synthetic exception in serve
         - influxdb python module not found
+        - \(MGR_ZABBIX_\)
```

Comment:
this does not work, see http://pulpito.ceph.com/kchai-2018-02-05_11:48:09-rados-wip-kefu-testing-2018-02-05-1650-distro-basic-mira/2155066/ . 

what we want to whitelist is cluster log like
```
[WRN] Health check failed: Failed to send data to Zabbix (MGR_ZABBIX_SEND_FAILED)
```

so `(MGR_ZABBIX_)` does not match.  but `(MGR_ZABBIX_` does. see
```
echo '[WRN] Health check failed: Failed to send data to Zabbix (MGR_ZABBIX_SEND_FAILED)' | egrep '\(MGR_ZABBIX'
[WRN] Health check failed: Failed to send data to Zabbix (MGR_ZABBIX_SEND_FAILED)
```

probably you could use `\(MGR_ZABBIX_` or `\(MGR_ZABBIX_SEND_FAILED\)` instead.